ratatool-sampling/README.md

```diff
@@ -13,6 +13,7 @@ For full details see [BigSample.scala](https://github.com/spotify/ratatool/blob/
 
 ## Usage
 
+### Command Line
 ```
 BigSampler - a tool for big data sampling
 Usage: ratatool bigSampler [dataflow_options] [options]
```

Comment:
do we need to add something here about Protobuf as well

Reply:
No, similar to BigDiffy, proto from CLI is not so simple.